### DISCUSSION TITLE
Activate scroll around borders

### DISCUSSION THREAD

**Discussion by dellyn** [idiotWu/smooth-scrollbar#478] [Q&A]
Hello! How is it possible to implement this functionality?
I have draggable fields inside smooth scrollbar and I want to have scroll when dragging inner field to container borders. Can i use some library features for that? Thank you!

<img width="564" alt="image" src="https://user-images.githubusercontent.com/54443742/171036728-13c10008-c8f8-4655-84f1-f7b678d99027.png">

**Comment by dolphin-wood** [comment]
I once created a pannable/draggable plugin in another project with custom events, check it out here: [keyframe-refiner/src/app/scrollbar-plugins.ts#L64-L118](https://github.com/keyframe-refiner/keyframe-refiner/blob/5497fe451411c612a4b0da501528b0c7e65f7106/src/app/scrollbar-plugins.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L64-L118).

👉 [Sample Usage](https://github.com/keyframe-refiner/keyframe-refiner/blob/5497fe451411c612a4b0da501528b0c7e65f7106/src/app/components/Cropper.svelte?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L70-L79)
